.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Not quite sure that we should add `.idea` to `.gitignore`. Personal dev tools-related git exclusions (related to OS, editor/IDE, etc.) should not pollute the shared `.gitignore`.

Reply:
@nurlansu Sorry for this. I just reverted that commit.